src/class-parsely-recommended-widget.php

```diff
@@ -12,10 +12,7 @@
 declare(strict_types=1);
 
 /**
- * This is the class for the recommended widget
- *
- * @category   Class
- * @package    Parsely_Recommended_Widget
```

Comment:
Why are we removing this kind of annotations?

Reply:
Because they aren't valid for class and function/method DocBlocks. 

`@category` is deprecated / removed from the draft list of [PHP DocBlock tags](https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc-tags.md), as `@package` can accept multiple levels (`@subcategory` is also deprecated / removed for the same reason).

@package is only typically useful until namespaces are used anyway, so we'd keep a correct `@package` in the file-level DocBlock (it's wrong in this file), and that's the only single `@package` tag that should be in a file.